frame/staking/src/pallet/impls.rs

```diff
@@ -754,13 +755,15 @@ impl<T: Config> Pallet<T> {
 	/// nominators.
 	///
 	/// This function is self-weighing as [`DispatchClass::Mandatory`].
-	pub fn get_npos_voters(maybe_max_len: Option<usize>) -> Vec<VoterOf<Self>> {
+	pub fn get_npos_voters(voter_bounds: DataProviderBounds) -> Vec<VoterOf<Self>> {
```

Comment:
```suggestion
	pub fn get_npos_voters(bounds: DataProviderBounds) -> Vec<VoterOf<Self>> {
```

for similar reasons; things that are implied should not be repeated in names. The function name already implies `voter` in this case.